ex03.ex

```diff
@@ -55,7 +55,14 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+  def odd_even([]), do: []
```

Comment:
### -2

Three problems here.

First, there's a bug: what does it return for

~~~ elixir
odd_even([1, :cat, 3, "wombat"])
~~~

Second: a case on a predicate function is really just an `if`.

Third: I'd rather you didn't use `if`, `cond`, or `case`.

~~~ elixir
  def odd_even([]), do: []
  def odd_even([ h | t ]), do: [ atom_for(h) | odd_even(t) ]

  defp atom_for(n) when Integer.is_odd(n),  do:  :odd
  defp atom_for(n) when Integer.is_even(n), do:  :even
~~~